examples/basic-integration/app/src/main/java/com/example/demoandroid/ClickToPaySuccessfulActivity.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.demoandroid
+
+import android.annotation.SuppressLint
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+
+class ClickToPaySuccessfulActivity : AppCompatActivity() {
+
+    companion object {
+        const val EXTRA_CLICK_TO_PAY_DATA = "EXTRA_CLICK_TO_PAY_DATA"
+    }
+
+    @SuppressLint("SetTextI18n")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_thank_you)
+
+//        val card = JSONObject(intent.getStringExtra(EXTRA_CLICK_TO_PAY_DATA)!!).toMap()
```

Comment:
@darwinmorocho-deuna si hay que dejar esta linea commented out?

Reply:
Es algo que tengo que revisar con @nelyj ya que en la respuesta de click to pay no vi que datos debo mostrar en la vista de success.

Deje la PR creada para avanzar con el review para hacer los ajustes lo mas pronto posible

Reply:
@darwinmorocho-deuna solo mostrra la misma y agregale el logo de click to pay. No esperes mas. Con este es suficiente para el demo app

Reply:
@esteban-deuna deje unicamente un texto que dice Click To Pay. Lunes por la mañana agrego la imagen de click to pay

Reply:
@esteban-deuna la vista de success lo deje unicamente con el texto  "Click To Pay Pago exitoso"